source/modules/Git.rst

```diff
@@ -92,22 +92,22 @@ With git, you work with *local* repositories and *remotes* that they are connect
       origin  https://github.com/UWPCE-PythonCert/IntroPython-2017.git (push)
 
    This shows that the local repo on my machine *originated* from the one in
-   the UWPCE-PythonCert gitHub account (it shows up twice, because there is
-   a shortcut for both fetch from and push to this remote)
+   the UWPCE-PythonCert GitHub account (it shows up twice, because there is
+   a shortcut for both fetch from and push to this remote).
 
 .. rst-class:: build
 .. container::
 
-    You can work on any project you wish to that has a public repository on Github. However, since you won't have permission to edit most projects directly, there is such a thing as *forking* a project.
+    You can work on any project you wish to that has a public repository on GitHub. However, since you won't have permission to edit most projects directly, there is such a thing as *forking* a project.
 
-    When you *fork* a repository, you make a copy of that repository in your own (Github) account.
+    When you *fork* a repository, you make a copy of that repository in your own (GitHub) account.
 
     When you have made changes that you believe the rest of the community will want to adopt, you make a *pull request* to the original project. The maintainer(s) of that project than have the option of accepting your changes, in which case your changes will become part of that project.
 
     This is how we will be working in this class. When you want feedback on your work, you will make a *pull request* to the instructors.
 
 
-Our class materials reside in a repository on *Github* in the *UWPCE-PythonCert* organization:
+Our class materials reside in a repository on *GitHub* in the *UWPCE-PythonCert* organization:
```

Comment:
I set up a demo repo that we'll copy for each "instance" of teh class:

https://github.com/UWPCE-PythonCert/DemoClassRepo

We can use that in the examples.